api/custom_auth/urls.py

```diff
@@ -4,16 +4,17 @@
 from custom_auth.views import (
     CustomAuthTokenLoginOrRequestMFACode,
     CustomAuthTokenLoginWithMFACode,
-    ThrottledUserViewSet,
+    FFAdminUserViewSet,
     delete_token,
 )
 
 app_name = "custom_auth"
 
-throttled_user_router = DefaultRouter()
-throttled_user_router.register(r"users", ThrottledUserViewSet)
+ffadmin_user_router = DefaultRouter()
+ffadmin_user_router.register(r"users", FFAdminUserViewSet)
```

Comment:
why has this changed from ThrottledUserViewSet to FFAdminUserViewSet?

Reply:
As discussed, a generic name is better for this class since it should be used to override any behavior related with FFAdminUser view